app/assets/src/components/views/PipelineViz/PipelineViz.jsx

```diff
@@ -719,9 +719,7 @@ class PipelineViz extends React.Component {
           minZoom={zoomMin}
           maxZoom={zoomMax}
           zoomSpeed={3}
-          ref={ref => {
```

Comment:
Do you know why setting this panZoomContainer like this does not work?
Is it null whenever you request current the first time?

Reply:
Yeah current is null if I use a callback ref